app/controllers/application_controller.rb

```diff
@@ -10,6 +16,12 @@ class ApplicationController < ActionController::Base
   include EnterprisesHelper
   include Spree::AuthenticationHelpers
 
+  # Helper for debugging strong_parameters
+  rescue_from ActiveModel::ForbiddenAttributesError, with: :print_params
+  def print_params
+    raise ActiveModel::ForbiddenAttributesError, params.to_s
+  end
+
```

Comment:
this can be removed now?

Reply:
It's pretty harmless, but I think I would wait until we test and merge of the branch and remove it afterwards. It'll be useful if we encounter any issues before merging.

Reply:
ok :+1: